src/main/java/io/getunleash/DefaultUnleash.java

```diff
@@ -1,27 +1,27 @@
-package no.finn.unleash;
+package io.getunleash;
```

Comment:
Don't you want to leave the old package in a deprecated state for a few releases? At lease for the classes that are most likely to be used, like `DefaulUnleash`?

Reply:
The plan is to make a major release with the package moved, but good point we should probably have the old package in place with a deprecation warning and just a redirect for the typical initialization stories. I'll update the PR.

Reply:
Since we're relocating the maven package coords as well, I think it's reasonable to expect our users to do a query-replace for their package imports. I'll remove the package move from this PR and do that in separate PR before we release 5.0 of the client. The stickiness does not require a new major version.

Reply:
The main problem comes from intermediate usages. For example if my company creates an internal helper library called helper:v1 that uses Unleash:v3 then it locks the app into not being able to upgrade their Unleash version to the next version until the helper library is upgraded.

Another example is with the recently merged in Chaos Monkey for Spring Boot support for Unleash.

Reply:
Once work around is the users could put both artifacts on the classpath and instantiate them twice. It would be good to avoid making users do that though

Reply:
Good point on internal helper libraries, we'll have a discussion for a proper package migration story then. 

Personally I would be fine with not being able to upgrade the backing library for a helper library until the helper library has upgraded, but I can see how that would impede progress/process as well. And I do agree with trying to avoid having our users double instantiate the UnleashClient.

I've removed the package migration away from this issue, so it's moot for this PR. But open for ideas over on #138